documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -131,6 +133,8 @@ A service is created to serve as the _bootstrap address_ for connection to the K
 A service is also created as the _external bootstrap address_ for external connection to the Kafka cluster using `nodeport` listeners.
 +
 The cluster CA certificate to verify the identity of the kafka brokers is also created with the same name as the `Kafka` resource.
++
+NOTE: Updates to listener configuration will trigger a rolling update of your Kafka cluster.
```

Comment:
I think the issue really says something else. It wants to say that when you scale up or down the Kafka cluster while using external listeners, it might roll all Kafka brokers. The `might` is important since it IMHO depends on the exactl confgiuration.

Reply:
I was going from my notes on the discussion (which were clearly wrong). 
Updated. I don't know if we need to expand, particularly around _depending on the configuration_.  It's a heads up, if nothing else.